### PR TITLE
[RHACS] RHACS Dashboard Updates

### PR DESCRIPTION
**Pull request by kcarmichael08**
Version(s):
- `rhacs-docs-3.71`
- `rhacs-docs`

Label:
`RHACS`

Issue: none

[Preview](https://openshift-docs-exljvm81l-kcarmichael08.vercel.app/openshift-acs/master/operating/view-dashboard.html)

### CODE REVIEW

**Comment by kcarmichael08** [comment]
Reopening

File: operating/view-dashboard.adoc [hunk 1]
```diff
@@ -7,104 +7,98 @@ include::modules/common-attributes.adoc[]
...
-* To display images at most risk, regardless of their active or inactive status, select *All images*. 
+[id="policy-violations-severity"]
+=== Policy violations by severity
 
-You can also perform the following actions in this widget:
+This widget shows the distribution of violations across severity levels for the Dashboard-filtered scope. Clicking a *severity level* in the chart takes you to the *Violations* page, filtered for that severity and scope. It also lists the three most recent violations of a *Critical* level policy within the scope you defined in the Dashboard filter. Clicking a specific violation takes you directly to the *Violations* detail page for that violation.
 
-* To view CVEs for an image, click on the image name.
-* To view a list of all the images at most risk, select *View all*.
+[id="imges-most-risk"]
```
Comment (jeana-redhat) [inline operating/view-dashboard.adoc:65]:
```suggestion
[id="images-most-risk"]
```

**Review by jeana-redhat — COMMENTED** [review]
Noticed a typo in an ID while scanning headings, and a table that I think is markdown rather than asciidoc.

File: operating/view-dashboard.adoc [hunk 2]
```diff
@@ -7,104 +7,98 @@ include::modules/common-attributes.adoc[]
...
+[options="header"]
+|====================================================================================================================
+| Counter     | Destination                                                           
+| Clusters    | Platform Configuration → Clusters                                      
+| Nodes       | Configuration Management → Application & Infrastructure → Nodes        
+| Violations  | Violations main menu                                                   
+| Deployments | Configuration Management → Application & Infrastructure → Deployments  
+| Images      | Vulnerability Management → Dashboard → Images                         
+| Secrets     | Configuration Management → Application & Infrastructure → Secrets      
+|====================================================================================================================
```
[the remark this answers is not in the document]
Comment (kcarmichael08) [reply operating/view-dashboard.adoc:33]: Ooo, sorry - I'm used to markdown from my last job so it didn't look weird to me - I think I've fixed it.

**Pull request merged**